packages/paypal-components/src/components/sqp-titled-section/sqp-titled-section.tsx

```diff
@@ -0,0 +1,62 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, h, Prop } from "@stencil/core";
+import { Spacing } from "../../global/mixins";
+import { getProps } from "../../utils/utils";
+import { PortalSectionView } from "../sqp-titled-section/sqp-portal-section-view";
+
+/**
+ * @uiName Titled Section
+ * @slots [{"name":"label", "title":"Label"},{"name":"content", "title":"Content"}]
+ * @exampleGroup Layout
+ * @example Titled Section - <sqm-titled-section text-align="center" label-margin="small" padding="none"><div slot="label"><h2>Section Label</h2></div><div slot="content"><p>Add your section content here!</p></div></sqm-titled-section>
+ */
+@Component({
+  tag: "sqm-titled-section",
```

Comment:
@00salmon noting that this folder is still in here - is there a reason? I remember we had talked about removing this